src/aof.c

```diff
@@ -2148,6 +2179,22 @@ off_t getAppendOnlyFileSize(sds filename) {
     return size;
 }
 
+off_t getIncrAppendOnlyFilesSize(aofManifest *am) {
+    off_t size = 0;
+
+    listNode *ln;
+    listIter li;
+
+    listRewind(am->incr_aof_list, &li);
+    while ((ln = listNext(&li)) != NULL) {
+        aofInfo *ai = (aofInfo*)ln->value;
+        serverAssert(ai->file_type == AOF_FILE_TYPE_INCR);
+        size += getAppendOnlyFileSize(ai->file_name);
```

Comment:
maybe that's another metadata to keep in the manifest?

Reply:
I understand what you mean. If we add `size` in manifest, do we need to check whether the size is the same as the actual size of the file when we load it?

There are still some questions about the manifest, that is, now we only have “name/type/seq” meta in manifest, do we need other meta, such as the timestamp of file creation, or other information (maybe we will use it in the future), because Once the manifest format is publish, it will not be modified later.

Reply:
yes, i'm not certain about my suggestion either.
i don't think we need do validate the size (as long as it's just used for progress report)
but anyway, it was just an idea, we can drop it.